src/components/Bale/styles.js

```diff
@@ -124,6 +124,36 @@ const styles = StyleSheet.create({
     color: Colors.white,
     fontSize: 17,
   },
+
+  // Modal styles
+  modalContainer: {
+    borderRadius: 4,
+    borderWidth: 2,
+    borderColor: Colors.primary,
+    backgroundColor: Colors.white,
+    height: 300,
+    width: 400,
```

Comment:
Can you use percentage instead?

Reply:
No. Since its the same modal for phones and tablets, theres no way to put percentages and get the size at a reasonable aesthetics with your suggestion